python/Ganga/GPIDev/Schema/Schema.py

```diff
@@ -402,7 +402,12 @@ def __len__(self):
     # item.isA(SimpleItem)
     def isA(self, _what):
 
-        from Ganga.GPIDev.Base.Proxy import stripProxy
+        try:
+            from Ganga.GPIDev.Base.Proxy import stripProxy
+        except ImportError:
+            def dummyfunc(arg):
+                return arg
+            stripProxy = dummyfunc
```

Comment:
Or rather this _was_ used in many places I think I swapped many of these out for better type ```isType``` checks.